### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by michaelputnam67**
Our initial commit adds iteration 0: Main-Page functionality.

### CODE REVIEW

**Comment by whiteheadol** [comment]
We are testing out our pull request template, and we want to see what it looks like when we reject a request.

**Comment by michaelputnam67** [comment]
Made adjustments to the pull request template.

File: src/main.js [hunk 1]
```diff
@@ -17,3 +15,51 @@ var currentCover;
...
+
+var currentCover = document.querySelector('.cover-image');
+currentCover.src = covers[getRandomIndex(covers)];
+var currentTitle = document.querySelector('.cover-title');
+currentTitle.innerText = titles[getRandomIndex(titles)];
+var firstDescriptor = document.querySelector('.tagline-1');
+firstDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+var secondDescriptor = document.querySelector('.tagline-2');
+secondDescriptor.innerText = descriptors[getRandomIndex(descriptors)];
+
```
Comment (whiteheadol) [inline src/main.js:28]: I think that this was a great way to pass Iteration 0, but I'm excited to see if we can condense it Iteration 1.

File: src/main.js [hunk 2]
```diff
@@ -17,3 +15,51 @@ var currentCover;
...
+
+
+
+
+
+
+
+
+
```
Comment (whiteheadol) [inline src/main.js:54]: Great! I like that we're saving these in case they are useful in future iterations.

File: src/main.js [hunk 3]
```diff
@@ -17,3 +15,51 @@ var currentCover;
 function getRandomIndex(array) {
```
Comment (michaelputnam67) [inline src/main.js:15]: Thank you Turing for giving us a head start.

**Comment by whiteheadol** [comment]
We tested this functionality in our console.

**Pull request merged**